docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Add a TensorFlow Lite Modular Service"
+linkTitle: "Add a TensorFlow Lite Modular Service"
+weight: 70
+type: "docs"
+description: "Add a custom MLModel modular-resource-based service which uses TensorFlow Lite to classify audio samples."
+tags: ["ml", "model training", "services"]
+# SMEs: Andrew Morrow
+---
+
+Viam provides an example [custom module](extend/modular-resources) written in C++ that extends the [ML model](/services/ml/) service to support audio classification using TensorFlow Lite.
+
+The example files can be found in the [Viam C++ SDK](https://github.com/viamrobotics/viam-cpp-sdk):
+
+- [`example_mlmodelservice_tflite.cpp`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/examples/modules/example_mlmodelservice_tflite.cpp) - a custom module that provides an example `MLModelService` instance which runs TensorFlow Lite models.
+- [`example_audio_classification_client.cpp`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/examples/mlmodel/example_audio_classification_client.cpp) - an example client instance which generates audio samples and invokes the `example_mlmodelservice_tflite` custom module to classify those samples.
+
+This tutorial walks you through everything necessary to start using these example files with your robot, including building the C++ SDK, procuring the necessary support files, configuring your robot and installing `viam-server`, and running the compiled binary example.
+
+## Build the C++ SDK
+
+To build the Viam C++ SDK, you will need a macOS or Linux computer.
+Follow the instructions below for your platform:
+
+{{< tabs >}}
+{{% tab name="macOS" %}}
+
+Follow the [Viam C++ SDK build instructions](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md) to build the SDK on your macOS computer using the `brew` package manager.
+
+While your specific build steps may differ slightly, your installation should generally resemble the following:
+
+1. Install all listed dependencies to support the Viam C++ SDK:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew install abseil cmake boost grpc protobuf xtensor pkg-config ninja buf
```

Comment:
I'm a little hesitant about re-iterating C++ SDK build instructions with this level of detail, because things are likely to diverge. How will we keep this list in sync as the C++ SDK requirements evolve, etc.? On the other hand, we want the tutorial to be as turnkey as possible so I understand the desire to provide detailed steps.

Reply:
Definitely valid. To get this working, I had to do some extra stuff not explicitly covered across BUILDING or README in the SDK repo, so I think there's value in being a bit explicit here.

Reply:
OK.